recipes/mosdepth/meta.yaml

```diff
@@ -1,36 +1,42 @@
-{% set name = "mosdepth" %}
 {% set version = "0.3.1" %}
 {% set sha256hash = "f6ecbff30396102ad2059faa0005453b51b27d31b960f421ef266f507da96816" %}
 
 package:
-  name: {{ name|lower }}
+  name: mosdepth
   version: {{ version }}
 
 source:
   - url: https://github.com/brentp/mosdepth/archive/v{{ version }}.tar.gz
     sha256: {{ sha256hash }}
-  - url: https://nim-lang.org/download/nim-1.2.0.tar.xz
-    sha256: 4e94583a373965821805e665e0a05f52fb610916676edb09148941415637c575
-    folder: nim_source
 
 build:
-  number: 1
+  number: 2
 
 requirements:
   build:
     - {{ compiler('c') }}
+    - git
+    - nim
   host:
-    - pcre
     - htslib
-  run:
+    - htslib >=1.10
```

Comment:
Good catch, makes sense.
I've re-added the constraint-less so that we get the version from pinning itself too (`conda-build` will only touch those entries without constraints when it applies the pinning; if we don't have those, the `htslib` dep might get out of sync with the pinning, e.g., use a future `htslib=1.13` when other packages haven't yet been rebuilt for that version; which would in turn mean you wouldn't be able to install `mosdepth` into the same environment with the other packages until those are rebuilt.)